helpers/http_api.js

```diff
@@ -58,7 +58,7 @@ var middleware = {
 			return next();
 		}
 		logger.error(`API error ${req.url}`, err.message);
-		console.trace(err);
+		console.error(err);
```

Comment:
This is quite different. We might want to add `.trace` to the list of allowed console methods.